wasm/validator/src/permission.rs

```diff
@@ -165,7 +165,7 @@ pub mod trigger {
     ) -> Result<bool> {
         FindTriggerById::new(trigger_id)
             .execute()
-            .map(|trigger| trigger.action().technical_account() == authority)
+            .map(|trigger| trigger.action().authority() == authority)
```

Comment:
We should back-propagate this change, the somewhat obscure name `technical_account` was chosen precisely because it was in the triggers RFC. Every deviation from that RFC was documented so should this one.

Reply:
so I should go to RFC and make a note of the change?

Reply:
No. You're good. I've opened a ticket and transferred to docs.